docs/guide/tutorial-6-4-wheels-robot.md

```diff
@@ -69,15 +64,12 @@ For example, in the case of an humanoid robot, the robot node would be typically
 In our case, the body box is obviously the better choice.
```

Comment:
60: The second step is to determine which [Solid](../reference/solid.md) node is the [Robot](../reference/robot.md) node (the root node).